src/Sylius/Bundle/ApiBundle/Command/Cart/ChangeItemQuantityInCart.php

```diff
@@ -13,18 +13,16 @@
 
 namespace Sylius\Bundle\ApiBundle\Command\Cart;
 
+use Sylius\Bundle\ApiBundle\Command\OrderItemIdAwareInterface;
 use Sylius\Bundle\ApiBundle\Command\OrderTokenValueAwareInterface;
 
 /** @experimental */
-class ChangeItemQuantityInCart implements OrderTokenValueAwareInterface
+class ChangeItemQuantityInCart implements OrderTokenValueAwareInterface, OrderItemIdAwareInterface
 {
     /** @var string|null */
     public $orderTokenValue;
 
-    /**
-     * @var string
-     * @psalm-immutable
-     */
+    /** @var string|null */
```

Comment:
I wonder if it shouldn't be:
```suggestion
    /** @var mixed|null */
```
as we have order item ids defined as integers, but there is a possibility to change that in end application to strings